packages/app/src/app/pages/Dashboard/index.tsx

```diff
@@ -45,13 +45,14 @@ export const Dashboard: FunctionComponent = () => {
   useEffect(() => {
     const isNewUser = browser.storage.get(NUOCT22);
 
-    if (isNewUser) {
-      // Open the create team modal for newly signed up users
-      actions.openCreateTeamModal();
+    if (isNewUser || isNewUser === false) {
+      if (isNewUser) {
```

Comment:
this looks weird :) what is the value/type of isNewUser? if it's a boolean, it looks like the first condition can be turned into a `isNewUser !== undefined`?

Reply:
It's either a boolean or undefined - boolean when it receives something from localStorage, undefined when it doesn't. I agree that it's weird... I don't think we use this before trials (unlike the `wnoct` variable) so we can make this better:

```
localStorage.setItem('NUOCT22', 'signup'); // for users that sign up themselves
localStorage.setItem('NUOCT22', 'invite'); // for users that sign up after invite
```

This way there is no confusion I think.